### DISCUSSION TITLE
Массовый фоновый вызов сервера при отрисовки страниц команд (баг платформы)

### DISCUSSION THREAD

**Discussion by DmitryChebotok** [alehinsasha/sketch1c#36] [General]
ГруппаКоманды

Происходит дерганье курсора при смене страниц если разные виды элементов в дереве

// p.s.
Баг проявляется на элементах где используется картинка из файла, 1С за каждой картинкой лезет на сервер и потом перерисовывает форму (встречал на собственных разработках)

**Comment by alehinsasha** [comment]
@DmitryChebotok а удалось как то этот баг платформы обойти (на ваших проектах) для внешней обработки?

**Comment by DmitryChebotok** [comment]
Нет, только частично убирая видимость у элементов, впринципе у вас так и реализовано. Либо отказаться от картинок и использовать библиотечные

Чеботок Дмитрий
+7(951)873-95-98

воскресенье, 16 мая 2021 г., 09:22 +0300 от ***@***.***  ***@***.***>:

**Comment by alehinsasha** [comment]
@DmitryChebotok спасибо. Тогда в обсуждения переведу ишуз, может позже решение найдется.
